include/cobs/stream.h

```diff
@@ -4,7 +4,13 @@
 #define COBS_STREAM_H_
 
 #include <stdbool.h>
+#include <version.h>
+
+#if ((KERNEL_VERSION_MAJOR) >= 3) && ((KERNEL_VERSION_MINOR) > 1)
```

Comment:
this check is not correct because it'd use the wrong code for 4.0

Reply:
good catch!